metron-platform/metron-hbase/metron-hbase-common/src/test/java/org/apache/metron/hbase/client/HBaseClientTest.java

```diff
@@ -149,7 +128,7 @@ public void testWrite() throws Exception {
     client.mutate();
 
     HBaseProjectionCriteria criteria = new HBaseProjectionCriteria();
-    criteria.addColumnFamily(WidgetMapper.CF_STRING);
+    criteria.addColumnFamily(Bytes.toString(cf));
```

Comment:
In the comments and test names in this class there are still references to "widgets".  Since the refactor this no longer makes sense. Would you be able to clean those out?

Reply:
Done